src/main/java/core/basesyntax/strategy/impl/NewYearDiscountService.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.strategy.DiscountService;
+
+public class NewYearDiscountService implements DiscountService {
+    public static final double NEWYEARDISCOUNT = 20.0d;

```

Comment:
```suggestion
    public static final double NEW_YEAR_DISCOUNT = 20.0;
```
same with other discounts